src/Microsoft.AspNetCore.Razor.Language/Legacy/CSharpCodeWriter.cs

```diff
@@ -651,5 +656,46 @@ public void Dispose()
             {
             }
         }
+
+        public struct StringLiteralWriter : IDisposable
```

Comment:
I was a little bit surprised too 😄 .
I have thought about helper method idea, but it means that this helper method in `RuntimeBasicWriter` have to have the knowledge of the C# syntax.